src/unity/python/turicreate/test/test_sframe.py

```diff
@@ -3444,6 +3444,20 @@ def test_builtins(self):
                 {'long': [12], 'unicode': [unicode('foo')]}))
 
         _assert_sframe_equal(sf, sf2)
+    
+    def test_add_column_nonSArray(self):
+        sf = SFrame()
+        sf.add_column([1,2,3,4],'x')
```

Comment:
Please add an assertion below, to make sure the behavior is correct and doesn't just fail silently. Assign the result to a variable, and compare that variable to an expected (constant) value using `_assert_sframe_equal`. And do this for each of the new tests.